backend cleanup fixmes

joka opened the pull request.

nidico commented:
Can you please rebase/merge?

pallix commented on src/adhocracy_core/adhocracy_core/authentication/__init__.py:
```diff
@@ -103,7 +103,8 @@ def get_tokenmanager(request: Request, **kwargs) -> ITokenManger:
     """Return not validated tuple with the X-User-Path/Token values."""
     user_url = request.headers.get('X-User-Path', '')
-    # FIXME find a proper solution, user_url/path as userid does not work
+    # TODO find a proper solution, useid should just be an identifier.
```
typo: useid -> userid

pallix commented on src/adhocracy_mercator/adhocracy_mercator/sheets/mercator.py:
```diff
@@ -436,7 +432,7 @@ def index_budget(resource: IResource, default) -> str:
     field, or "above_50000" if the total budget value is more than 50,000 euro.
     """
-    # FIXME: Why is finance '' in the first pass of that function
+    # TODOE: Why is finance '' in the first pass of that function
```
typo

pallix commented:
When running the protractor acceptance tests locally, there is a lot of tests failing on this branch, specially the one concerning the ratings.

joka commented:
> When running the protractor acceptance tests locally, there are a lot of tests failing on this branch, specially the one concerning the ratings.

Are these tests stable?

pallix commented:
When runned locally yes.

joka commented:
i fixed one regression regarding subscriber registration and renamed some stuff prevent this kind of errors in the future.

All tests pass on my local machine.

src/mercator/tests/acceptance/test_mercator_filter.py did only  pass if run manually, but this is an unrelated issue.
